.github/workflows/helm-community.yml

```diff
@@ -86,7 +88,7 @@ jobs:
           helm install ${{ matrix.name }} . \
             --set global.search.sharedSecret="$(openssl rand -hex 24)" \
             --set global.known_urls=http://localhost \
-            --set global.alfrescoRegistryPullSecrets=regcred \
+            --set global.alfrescoRegistryPullSecrets=${{ env.REGISTRY_SECRET_NAME }} \
```

Comment:
maybe is better if you move the condition to the env declaration so it will be empty here as well when forks